dn1/src/dn1.jl

```diff
@@ -0,0 +1,155 @@
+module dn1
+
+export RedkaMatrika, sor
+
+import Base: getindex, setindex!, size, *, -
+
+"""
+    RedkaMatrika(V, I)
+
+podatkovni tip za redke matrike. Polja sta matriki velikosti nxm z neničelnimi elementi, kjer je m<=n ter matrika ki jo opisuje velikosti nxn. 
+V mattriki V so zapisane vrednosti neničelnih elementov v posameznih vrsticah. V matriki I so zapisani stolcpi istoležnih vvrednosti v posameznih vrsticah
+"""
+mutable struct RedkaMatrika
```

Comment:
Če bi namesto matriki za V in I vzeli gnezdene vektorje (`Vector{Vector}`), bi bilo lažje dodajati nove elemente in bi lahko odstranili `mutable`. Moram spremeniti navodila.